<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get Info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/Resources/favicon.ico</normaloff>:/icons/Resources/favicon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="searchParametersTab">
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/icons/Resources/1377003707_bullet_find.png</normaloff>:/icons/Resources/1377003707_bullet_find.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Search parameters</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="activeResourceWidget" native="true"/>
          <widget class="QWidget" name="searchParametersWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Location</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QLineEdit" name="countryField"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>City</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLineEdit" name="cityField"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Education</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>University</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLineEdit" name="univercityField"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Skills and Expertise</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="1" column="0">
                <widget class="QLineEdit" name="knowledgeField"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Knowledge</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="searchBtn">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="resultTab">
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/icons/Resources/1377003918_page_find.png</normaloff>:/icons/Resources/1377003918_page_find.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Result</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QStackedWidget" name="resultStackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="resultListViewPage">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTableView" name="resultListView">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="resultProgressBarPage">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QProgressBar" name="resultProgressBar">
               <property name="maximum">
                <number>0</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="bookMrkBtn">
                <property name="text">
                 <string>Add to contacts list</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="DetailsResultWidget" name="detailsResultWidget" native="true"/>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="openProfile">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Open profile in browser</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchHistoryTab">
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/icons/Resources/1377004004_clock_history_frame.png</normaloff>:/icons/Resources/1377004004_clock_history_frame.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Search history</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="historyTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::NoDragDrop</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="filterWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>From</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QDateEdit" name="dateFrom"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QDateEdit" name="dateTo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Knowledge</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="knowledgeCmb"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>University</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="universityCmb"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Resource name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="resourceNameCmb"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Location</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>City</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cityCmb"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Country</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="countryCmb"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
          <zorder>groupBox_2</zorder>
          <zorder></zorder>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionEnable_filter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/icons/Resources/1377003296_cog.png</normaloff>:/icons/Resources/1377003296_cog.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionUpdate_result">
   <property name="text">
    <string>update result</string>
   </property>
  </action>
  <action name="actionEnable_filter">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/icons/Resources/1377003410_filter.png</normaloff>:/icons/Resources/1377003410_filter.png</iconset>
   </property>
   <property name="text">
    <string>Enable filter</string>
   </property>
  </action>
  <action name="actionOpenSearchResults">
   <property name="text">
    <string>Open search results</string>
   </property>
  </action>
  <action name="actionDeleteHistoryRecord">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DetailsResultWidget</class>
   <extends>QWidget</extends>
   <header>Gui/Widgets/DetailsResultWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionSearch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
